Reneiski_Homework1/ReneiskiSelenium11/Common/Extensions/ExtensionMethods/ConvertText.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace ReneiskiSelenium11.Common.Extensions.ExtensionMethods
+{
+    public static class ConvertText
```

Comment:
use extension methods here (and if you use it, it's better to name class like StringExtension)

Reply:
Fixed